assets/css/styles.css

```diff
@@ -75,10 +75,28 @@ tr:nth-child(even) {
 
 /*an option for a svg to be used directly*/
 .svg {
-    background: url("http://fofwebdesign.co.uk/template/_testing/test/svg/page-bg.svg") no-repeat;
     background-size: 100%;
     height: 100vh;
     width: 100%;
-    background-position: 50px 50px;
+    background: url("http://fofwebdesign.co.uk/template/_testing/test/svg/page-bg.svg") no-repeat 50px 50px;
```

Comment:
Is this equivalent to the two deleted lines above?

Reply:
that is correct . same behavior